docs/components/movement-sensor/imu/imu-wit.md

```diff
@@ -83,5 +83,5 @@ Edit and fill in the attributes as applicable.
 
 Name | Type | Inclusion | Description
 ---- | ---- | --------- | -----------
-`serial_path` | string | **Required** | The name of the port through which the sensor communicates with the computer.
-`serial_baud_rate` | int | Optional | The rate at which data is sent from the sensor, between `9600` and `115200`. The default rate will work for all models. *Only the HWT901B can have a different serial baud rate.* Refer to your model's data sheet. <br> Default: `115200`
+`serial_path` | string | **Required** | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyS0`, to its device file, such as <file>/dev/ttyS0</file>. If you omit this attribute, Viam will attempt to automatically detect the path.
+`serial_baud_rate` | int | Optional | The rate at which data is sent from the sensor over the serial connection. Valid rates are `9600` and `115200`. The default rate will work for all models. *Only the HWT901B can have a different serial baud rate.* Refer to your model's data sheet. <br>Default: `115200`

```

Comment:
Baud rates are not a range, but a selection of only two valid values. [[source](https://github.com/viamrobotics/rdk/blob/7c72f44c6e9f7ddabc9615c81a3999f9d70a6160/components/movementsensor/imuwit/imu.go#L14-L15)]